### PR TITLE
Bug1646100 5.7

### PR DESCRIPTION
**Pull request by laurynas-biveinis**
http://jenkins.percona.com/job/mysql-5.7-param/584/

@robgolebiowski, please review as well

### CODE REVIEW

**Review by percona-ysorokin — APPROVED** [review]
LGTM

File: sql/binlog.h [hunk 1]
```diff
@@ -949,6 +948,8 @@ class MYSQL_BIN_LOG: public TC_LOG
       @retval !=0    Error
   */
   int get_gtid_executed(Sid_map *sid_map, Gtid_set *gtid_set);
+private:
+  void publish_global_status_variables(void) const;
```
Comment (robgolebiowski) [inline sql/binlog.h]: maybe I would just changed the name of this function to publish_global_snapshot_status_variable.

**Comment by robgolebiowski** [comment]
I added only one comment. LGTM

**Comment by laurynas-biveinis** [comment]
@robgolebiowski, please take a look at https://github.com/percona/percona-server/pull/1304 too for 5.6

File: sql/binlog.h (hunk 1 above)
Comment (laurynas-biveinis) [reply sql/binlog.h]: Hmm indeed not the best name. publish_position_for_global_status ?

Comment (robgolebiowski) [reply sql/binlog.h]: I think this is better than the previous one. Just note that apart from position you are also publishing binlog_global_snapshot_file. Maybe publish_position_for_global_snapshot_status, but publish_position_for_global_status is good too.

Comment (laurynas-biveinis) [reply sql/binlog.h]: Ah, "coordinates" then (a pair of name, position), making  it publish_coordinates_for_global_status ?

Comment (robgolebiowski) [reply sql/binlog.h]: Agreed :+1:

**Comment by laurynas-biveinis** [comment]
Updated method name

**Comment by robgolebiowski** [comment]
LGTM

**Pull request merged**